### PR TITLE
Rename the downloaded file when exporting or converting case

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

File: src/components/utils/caseUtils.ts [hunk 1]
```diff
@@ -11,21 +11,30 @@ import {
...
+} from '@gridsuite/commons-ui';
 import { useCallback, useState } from 'react';
 
-const downloadCases = async (uuids: string[]) => {
-    for (const uuid of uuids) {
-        const result = await downloadCase(uuid);
-        let name = await getCaseOriginalName(uuid);
+const downloadCases = async (selectedCases: ElementAttributes[]) => {
+    for (const selectedCase of selectedCases) {
+        const result = await downloadCase(selectedCase.elementUuid);
```
Comment (SlimaneAmar) [inline src/components/utils/caseUtils.ts:23]: the fetch `downloadCase` need to get also the format so as not to call the `getCaseOriginalName`
In an another US

File: src/components/utils/caseUtils.ts [hunk 2]
```diff
@@ -65,12 +74,16 @@ export function useDownloadUtils() {
                 .get('Content-Disposition')
                 .split('filename=')[1];
             filename = filename.substring(1, filename.length - 1); // We remove quotes
+            const fileExtension = filename.split('.').pop();
```
Comment (SlimaneAmar) [inline src/components/utils/caseUtils.ts]: Add an enpoint like` network-converion-server` with filename (without extension)
and get extension from the back

Comment (souissimai) [reply src/components/utils/caseUtils.ts]: we do not have extension, we have only format which is not the same: 
Example 
 format : MATPOWER
 extension : mat
I think that the actual solution to get format is to extract it from original case name

Comment (souissimai) [reply src/components/utils/caseUtils.ts]: done

File: src/components/utils/caseUtils.ts [hunk 3]
```diff
@@ -57,20 +66,21 @@ export function useDownloadUtils() {
...
+                caseElement.elementName,
                 format,
                 formatParameters,
                 abortController
             );
             let filename = result.headers
                 .get('Content-Disposition')
                 .split('filename=')[1];
-            filename = filename.substring(1, filename.length - 1); // We remove quotes
+
```
Comment (SlimaneAmar) [inline src/components/utils/caseUtils.ts]: Need to remove quotes

File: src/components/menus/content-contextual-menu.jsx [hunk 4]
```diff
@@ -350,6 +351,14 @@ const ContentContextualMenu = (props) => {
                     nameItem: elementName[1],
                 });
             }
+            //update selected elements if element is renamed
+            const updatedSelectedElements = selectedElements.map((element) => {
```
Comment (SlimaneAmar) [inline src/components/menus/content-contextual-menu.jsx:355]: Remove callback

File: src/components/utils/caseUtils.ts (hunk 3 above)
Comment (souissimai) [reply src/components/utils/caseUtils.ts]: done

File: src/components/menus/content-contextual-menu.jsx (hunk 4 above)
Comment (souissimai) [reply src/components/menus/content-contextual-menu.jsx:355]: done

**Review by SlimaneAmar — APPROVED** [review]

**Pull request merged**